remote.pro

```diff
@@ -506,6 +506,7 @@ win32 {
 
 DISTFILES += \
     components/FpsItem.qml \
+    components/media_player/ui/EPGItemView.qml \
```

Comment:
Does this belong in here? I'm still not very experienced with qml, so let's ask @martonborzak 
My FpsItem.qml might be wrong here as well...

Reply:
solved for me